invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/components/Header/Header.js

```diff
@@ -48,7 +55,23 @@ export default class Header extends Component {
                 Backoffice
               </Dropdown.Item>
               <Dropdown.Divider />
-              <Dropdown.Item>Sign out</Dropdown.Item>
+              {sessionManager.isUserLoggedIn() ? (
```

Comment:
I noticed in the last lines we start to introduce conditional rendering when the user is loggedIn or not. I am not sure to what extend these will be used. It might be just the to right button on the toolbar which makes it ok, or it might change the logic of the whole page which makes it not ok to write all possible conditions. Take into consideration that apart from logged in logged out, we might have different renders for each role. So food for thought!

Reply:
Yes I agree with you but this is something that will be addressed when we handle the frontsite. This was just to have a basic login/logout functionality.